river/bandit/thompson.py

```diff
@@ -15,11 +15,15 @@ class ThompsonSampling(bandit.base.Policy):
     distribution, but should rather be defined in the policy parametrization. In other words, you
     should do this:
 
-    >>> policy = ThompsonSampling(dist=proba.Beta(1, 1), seed=42)
+    ```
```

Comment:
nit: I think ```py is better :)

Reply:
I don't know why, but it didn't work, at least locally. "py" was rendered in the docstring :/

Reply:
No worries. The docs looks cool now!